package br.com.dio.util.model;

public enum GameStatusEnum {

  NON_STARTED("Não iniciado"),
  INCOMPLETE("Incompleto"),
  COMPLETE("Completo");

  private String label;
  GameStatusEnum(String label) {
    this.label = label;
  }
  
  public String getLabel() {
    return label;
  }



}
